components/Banner/tests/snapshot/MDCBannerSnapshotTests.m

```diff
@@ -19,9 +19,12 @@
 #import "MaterialTypographyScheme.h"
 
 static NSString *const kBannerShortText = @"tristique senectus et";
+static NSString *const kBannerMiddleLengthText =
+    @"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.";
 static NSString *const kBannerLongText =
     @"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.";
 static const CGFloat kBannerContentPadding = 10.0f;
+static const CGFloat kBannerLargeContentPadding = 30.0f;
```

Comment:
Nit: Please add comments to explain what this is used for.

Reply:
Addressed in 09ce09c952e7ca194c3dcee1a4816819aef2cbbe.